packages/@webex/internal-plugin-conversation/test/unit/spec/conversation.js

```diff
@@ -135,7 +134,7 @@ describe('plugin-conversation', () => {
       it('should convert a "us" cluster to WEBEX_CONVERSATION_DEFAULT_CLUSTER cluster', async () => {
         await webex.internal.conversation.getUrlFromClusterId({cluster: 'us'});
 
-        sinon.assert.calledWith(webex.internal.services.getServiceFromClusterId, {clusterId: 'urn:TEAM:us-east-2_a:identityLookup'});
+        sinon.assert.calledWith(webex.internal.services.getServiceFromClusterId, {clusterId: 'urn:TEAM:us-east-1_int13:identityLookup'});
```

Comment:
why this change ?

Reply:
This test was failing so fixed it with this change.